frontend/libs/studio-components-legacy/src/components/StudioCodelistEditor/StudioCodeListEditor.test.tsx

```diff
@@ -344,6 +344,74 @@ describe('StudioCodeListEditor', () => {
     });
   });
 
+  describe('onCreateTextResource', () => {
+    it('Calls the onCreateTextResource callback with the new text resource when a help text is changed and there is no text resource linked to it', async () => {
```

Comment:
Her mangler det også tester for ledetekst og beskrivelse. Du kan bruke [`it.each`](https://jestjs.io/docs/api#testeachtablename-fn-timeout) til å kjøre samme test for hver `CodeListItemTextProperty`.